bentoml/_internal/io_descriptors/__init__.py

```diff
@@ -1,4 +1,8 @@
+from __future__ import annotations
+
+from .base import from_spec
 from .base import IODescriptor
+from .base import IO_DESCRIPTOR_REGISTRY
```

Comment:
I don't think we should expose the registry here.

Reply:
I think this was left over from when I was implementing from_spec here; I'll remove this.